release-team/role-handbooks/release-notes/README.md

```diff
@@ -173,6 +173,29 @@ To update an entry in this section the following steps must be performed:
 
 ## Release Cycle Milestone Activities:
 
+### Week 0 
+
+Create an internal schedule for the release notes team to follow based on the release timeline. For example, for the 
+[v1.31 release](https://github.com/kubernetes/sig-release/tree/master/releases/release-1.31), create a table to track
+Release, Branch Created Day, Week of Release, PR merge deadline, and Release Notes Assignee. 
+
+| **Release**                                                                               | **Branch Created Day**                       | **Week**                                                              | **PR merge deadline** | **Release Notes Assignee** |
+| -------------------------------------------------------------------------------------- | ----------------------------- | --------------------------------------------------------------------- | ------- |----------------------------|
```

Comment:
Wondering if it would be easier to read if Branch created day is placed before PR merge deadline, and maybe they should be next to each other. Just so the days flow chronologically when reading

Reply:
Good idea!